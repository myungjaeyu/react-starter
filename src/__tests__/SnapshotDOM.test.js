import React from 'react';

import renderer from 'react-test-renderer';
import SnapshotDOM from '../common/SnapshotDOM';


test('snapshot test', () => {

    const dom = renderer.create(
        <SnapshotDOM href='https://adshot.asia' title='Adshot' />
    );


    const obj = dom.toJSON();

    expect(obj).toMatchSnapshot();

});