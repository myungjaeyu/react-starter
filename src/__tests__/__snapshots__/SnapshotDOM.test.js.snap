// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<div>
  <a
    href="https://adshot.asia"
  >
    <h1>
      Adshot
    </h1>
  </a>
</div>
`;
